crates/models/llama/src/lib.rs

```diff
@@ -52,39 +63,54 @@ impl KnownModel for Llama {
 
         // model-global weights
         let wte = tl.load("tok_embeddings.weight")?;
-        let norm = tl.load("norm.weight")?;
-        let output = tl.load("output.weight")?;
+
+        let backend = if params.should_offload(0) {
+            Backend::Gpu
+        } else {
+            Backend::Cpu
+        };
+
+        let norm = tl.offload("norm.weight", backend)?;
```

Comment:
Ah, I see, the `offload` is a bit more convenient. I think `load("norm.weight")?.transfer_to(backend)?` is an acceptable compromise to reduce the complexity of the loader. (PyTorch uses `to`, which could also work, but I'm not sure if that's too short for Rust.)

Reply:
`to` could be confusing as the offloading could change the underlying tensor structure, which `to` in pytorch doesn't do.
`transfer_to` seams fine.